pkgs/applications/graphics/image_optim/default.nix

```diff
@@ -15,18 +16,31 @@
 with lib;
 
 let
-  optionalDepsPath = []
-    ++ optional withPngcrush pngcrush
+  optionalDepsPath = optional withPngcrush pngcrush
     ++ optional withPngout pngout
```

Comment:
Maybe we should set this off by default to beable to build the package on ofborg.